### PR TITLE
Start implementation of single endpoint with all info

### PR DESCRIPTION
**Pull request by mattwr18**
fixes#8

### CODE REVIEW

File: app/views/charity_overview/_charity.json.jbuilder [hunk 1]
```diff
@@ -0,0 +1 @@
+json.extract! charity_overview, :id, :regno, :subno, :name, :desc, :orgtype, :gd, :aob, :aob_defined, :nhs, :ha_no, :corr, :add1, :add2, :add3, :add4, :add5, :postcode, :phone, :fax, :lon, :lat, :created_at, :updated_at
```
Comment (tansaku) [inline app/views/charity_overview/_charity.json.jbuilder]: do we want to add :desc in here?

File: app/views/charity_overview/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @charities.each do |charity| %>
+      <tr>
+        <td><%= charity.regno %></td>
+        <td><%= charity.subno %></td>
+        <td><%= charity.name %></td>
+        <td><%= charity.orgtype %></td>
```
Comment (tansaku) [inline app/views/charity_overview/index.html.erb]: do we need a charity.desc in here too?

Comment (mattwr18) [reply app/views/charity_overview/index.html.erb]: that is probably sensible... I don't know how many people use the index.html.erb, but I can't think of a solid reason why not to add it

Comment (tansaku) [reply app/views/charity_overview/index.html.erb]: but not a reason to prevent this all being merged and deployed ...

**Pull request merged**